apple/RNGestureHandlerDirection.h

```diff
@@ -1,8 +1,33 @@
 #import <Foundation/Foundation.h>
 
+#ifndef RNGestureHandlerDirection_h
+#define RNGestureHandlerDirection_h
+
 typedef NS_ENUM(NSInteger, RNGestureHandlerDirection) {
   RNGestureHandlerDirectionRight = 1,
   RNGestureHandlerDirectionLeft = 2,
   RNGestureHandlerDirectionUp = 4,
   RNGestureHandlerDirectionDown = 8,
+  RNGestureHandlerDirectionUpLeft = 16,
+  RNGestureHandlerDirectionUpRight = 32,
+  RNGestureHandlerDirectionDownLeft = 64,
+  RNGestureHandlerDirectionDownRight = 128,
```

Comment:
I believe this doesn't work - we use logical `or` to combine directions, so setting `Up | Left` on JS side will result in `6`, while here `16` is expected. Or am I missing something?

Reply:
Now you've got me thinking. I agree that it shouldn't, but somehow it does. I'll check why and then get back here.

Reply:
Okay, in case of diagonals you're right, I've changed that in [a9bb4f6](https://github.com/software-mansion/react-native-gesture-handler/pull/3028/commits/a9bb4f69fd6ddeef5c791b37434cfe2f0145684b)